<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>801</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="mainHeader">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>801</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>40</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Image Classify GUI</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>70</y>
      <width>800</width>
      <height>471</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <property name="font">
      <font>
       <family>Times New Roman</family>
       <pointsize>12</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <attribute name="title">
      <string>Classify</string>
     </attribute>
     <widget class="QLabel" name="classifyLabel">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>40</y>
        <width>151</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Choose File to Classify:</string>
      </property>
     </widget>
     <widget class="QPushButton" name="chooseFile">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>70</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Select File</string>
      </property>
     </widget>
     <widget class="QLabel" name="imageLabel">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>22</y>
        <width>461</width>
        <height>231</height>
       </rect>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="text">
       <string>Select Image</string>
      </property>
      <property name="scaledContents">
       <bool>false</bool>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QPushButton" name="classifyButton">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>150</y>
        <width>141</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Run Classification</string>
      </property>
     </widget>
     <widget class="QLabel" name="resultsDataLabel">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>315</y>
        <width>701</width>
        <height>81</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <property name="font">
      <font>
       <family>Times New Roman</family>
       <pointsize>12</pointsize>
       <strikeout>false</strikeout>
      </font>
     </property>
     <attribute name="title">
      <string>Retrain</string>
     </attribute>
    </widget>
   </widget>
   <widget class="QLabel" name="namesHeader">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>60</y>
      <width>401</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>By: Cade Baker, Megan Thomas, Brendan Nahed</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>801</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionExit"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
